packages/flutter_bloc/example/lib/main.dart

```diff
@@ -2,12 +2,15 @@ import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 
 void main() {
-  Bloc.observer = AppBlocObserver();
+  Bloc.observer = const AppBlocObserver();
   runApp(const App());
 }
 
 /// Custom [BlocObserver] that observes all bloc and cubit state changes.
 class AppBlocObserver extends BlocObserver {
+  /// Custom [BlocObserver] that observes all bloc and cubit state changes.
```

Comment:
Can we create a template and use a macro here to avoid duplicating the documentation?